src/wallet/test/availablecoins_tests.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
```suggestion
// file COPYING or https://www.opensource.org/licenses/mit-license.php.
```

Reply:
huh, interesting. seems like we have a mix of http and https throughout the codebase in the license header (mental note to fix this in a follow up)